azurerm/resource_arm_sql_database.go

```diff
@@ -257,3 +287,20 @@ func validateArmSqlDatabaseEdition(v interface{}, k string) (ws []string, errors
 	}
 	return
 }
+
+func validateArmSqlDatabaseCreateMode(v interface{}, k string) (ws []string, errors []error) {
```

Comment:
(if we switch to using the `validation.StringInStrings` and the enum's/constants above - I think we can remove this?)